LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please do not add this file. You should not modify anything outside of the recipes folder for the package you are building.